### PR TITLE
Refactoring/cv2 4944 cleanup code

### PR DESCRIPTION
**Pull request by amoedoamorim**
## Description

- Fixes `article?.` abuse in `ArticleForm.js`
- Fixes New standalone fact-check has no report status icon
- Fixes report status not reflected correctly in Slideout
- Fixes failed prop type warnings
- Fixes required key prop warning

References: CV2-4944

## Type of change

- [x] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

Manually

## Things to pay attention to during code review

Please describe parts of the change that require extra attention during code review, for example:

- File FFFF, line LL: This refactoring does this and this. Is it consistent with how it’s implemented elsewhere?
- Etc.

## Checklist

- [x] I have performed a self-review of my own code
- [x] I've made sure my branch is runnable and given good testing steps in the PR description
- [ ] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [x] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [ ] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

File: src/app/components/cds/media-cards/ItemReportStatus.js [hunk 1]
```diff
@@ -35,10 +41,18 @@ const ItemReportStatus = ({
     );
   };
 
+  const handleGoToReport = () => {
+    if (projectMediaDbid) {
+      console.log('projectMediaDbid:', projectMediaDbid); // eslint-disable-line no-console
```
Comment (caiosba) [inline src/app/components/cds/media-cards/ItemReportStatus.js:46]: @amoedoamorim please remove this before merging

File: src/app/components/search/SearchResultsCards/ArticleCard.js [hunk 2]
```diff
@@ -92,21 +98,27 @@ ArticleCard.defaultProps = {
...
+  projectMediaDbid: null,
   publishedAt: null,
 };
 
 ArticleCard.propTypes = {
   title: PropTypes.string.isRequired,
   summary: PropTypes.string,
   url: PropTypes.string,
-  date: PropTypes.number.isRequired, // Timestamp
+  //
```
Comment (caiosba) [inline src/app/components/search/SearchResultsCards/ArticleCard.js:109]: Please remove this empty comment too

**Review by caiosba — APPROVED** [review]
Thanks @amoedoamorim ! Just a couple small clean ups before merging!

PS: Alex and I pair-reviewed most of it.

File: src/app/components/search/SearchResultsCards/ArticleCard.js (hunk 2 above)
Comment (amoedoamorim) [reply src/app/components/search/SearchResultsCards/ArticleCard.js:109]: oopsie!

File: src/app/components/cds/media-cards/ItemReportStatus.js (hunk 1 above)
Comment (amoedoamorim) [reply src/app/components/cds/media-cards/ItemReportStatus.js:46]: oopsie!

**Pull request merged**